server/sonar-server/src/main/java/org/sonar/server/ws/ServletRequest.java

```diff
@@ -73,15 +73,21 @@ protected String readParam(String key) {
 
   @Override
   protected InputStream readInputStreamParam(String key) {
+    Part part = readPart(key);
+    return (part == null) ? null : part.getInputStream();
+  }
+
+  @CheckForNull
+  public Part readPart(String key) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1161)